src/site/content/en/handbook/content-checklist/index.md

```diff
@@ -48,7 +48,7 @@ This is a checklist for web.dev content reviews. It includes critical high-level
 1. Are all links to web.dev pages and assets relative?
 
 ## Mechanics
-1. Is the text free of spelling and capitalization errors? (Check the [word list](/word-list).)
+1. Is the text free of spelling and capitalization errors? Check the [word list](/word-list).
 1. Are [dashes and hyphens](/handbook/grammar/#dashes-and-hyphens) used correctly?
 1. Are titles and headings in sentence case with no terminal period?
```

Comment:
what's a terminal period? Might be simpler to say "Are they free of punctuation?"

Reply:
```suggestion
1. Are titles and headings in sentence case with no period at the end?
```
Well, that's not really accurate. They might need internal punctuation (commas, hyphens), for example. Suggested a revision.

Reply:
Fixed in Google Doc